processing/src/main/java/io/druid/segment/incremental/IncrementalIndex.java

```diff
@@ -1379,10 +1407,31 @@ public long getMaxTimeMillis()
       );
     }
 
+    private Stream<IncrementalIndexRow> timeAndDimsOrderedConcat(
+        final Collection<Deque<IncrementalIndexRow>> rowGroups,
+        final boolean descending
+    )
+    {
+      if (descending) {
```

Comment:
The descending code path is never used, so please remove it. I think in general we want to avoid unused code, since it can confuse readers, and it can have 'sleeper' bugs that awake if someone ever tries to use the code.

Reply:
Oops, good point, my copy pasta is showing 🍝 :+1: